src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -58,13 +56,17 @@
   
   <UsingTask TaskName="GenerateRuntimeGraph" AssemblyFile="$(_generateRuntimeGraphTask)"/>
 
-  <Target Name="GenerateRuntimeJson" Condition="'$(AdditionalRuntimeIdentifiers)' != ''">
+  <Target Name="GenerateRuntimeJson" BeforeTargets="Build">
```

Comment:
Why is this hooked onto `BeforeTargets="Build"`? The `GenerateRuntimeGraph` task necessary for the execution of this target is a build output.

Reply:
Ah, yes. I've changed it to `AfterTargets`.